test/e2e/network_test.go

```diff
@@ -233,6 +231,12 @@ func testNorthSouthNetworking(t *testing.T) {
 	testCtx, err := NewTestContext()
 	require.NoError(t, err)
 
+	// Ignore the application ingress load balancer test for None and vSphere platforms as it has to be created manually
+	// https://docs.openshift.com/container-platform/4.5/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer.html#nw-using-load-balancer-getting-traffic_configuring-ingress-cluster-traffic-load-balancer
```

Comment:
lets not link to 4.5 docs here, updating it to 4.9 would be preferred if 4.10 docs arent up yet

Reply:
Another suggestion is replacing an explicit version with `latest` so it's always up to date

Reply:
I'd rather stick with `4.9`. The content may change in the future and there is a possibility that the `latest` documentation is no longer relevant to this context.

Reply:
IMO, the link points to general documentation not necessarily related to this issue and should be removed. I [added](https://github.com/openshift/windows-machine-config-operator/pull/896#discussion_r806338505) it back per @selansen  request.